### DISCUSSION TITLE
Login Fails on Fresh install of Linux; same user / pw works fine on old version of linux

### DISCUSSION THREAD

**Discussion by pgmer6809** [community/community#59841] [Programming Help]
I have a github developer account with one of those long, github generated passwords.
On my old install of linux, the user name and password work fine. I can paste the PW into either Firefox or Chromium.
I can also login from the terminal
On a fresh install of Linux login using the terminal still works but login using either of the browsers does not work.
I get a 'Incorrect Username or Password' error.
However on my other more personal account, with a 'normal' 10 char password, login through either browser works ok.
How do make it so that I can login to the developer account through at least one browser from both the old and the new installs of Linux.

**Comment by queenofcorgis** [comment]
Thanks for posting in the GitHub Community, @pgmer6809 ! 

We’ve moved your post to our new Programming Help 🧑‍💻 category, which is more appropriate for this type of discussion and may be more helpful. 

Please review our [guidelines](https://github.com/orgs/community/discussions/51384) about the Programming Help category for more information.

**Comment by davea42** [comment]
If the following is old news to you then sorry. Just trying to help.

It is very easy to manage the copy part of copy/paste and manage to only pick up part of the long
password.  If you have too narrow a window (where the pwd shows) and drag the mouse across
to the visible end that will  not suffice. Put the pwd in a vertically small  but wide-enough window.  

Of course I know this because at one point developer password things doubled in size and
the generated password did not fit in the window showing the new password.  
So I could not use that generated password, had to generate  a new one and record it
properly.  I was rather distressed for a few minutes...

Hope this helps.
